### PR TITLE
Fix/remove bootstrap references

### PR DESCRIPTION
**Pull request by vinicius77**
1.Describe the changes made:

    Removed Bootstrap references from this page,
    Re-worked the content to make sense given the new context without Bootstrap,
    Identified other places in the JavaScript section that referenced Bootstrap and removed them.
    Removed the unnecessary extra div elements originally used to apply the Bootstrap classes

2. If this PR is related to an open issue please reference it with the # operator and the issue number below:

Remove Bootstrap mentions in the React portion of Javascript #21945

### CODE REVIEW

**Comment by vinicius77** [comment]
Hi @marvingay and @twalton83, 

I have created this new branch in order to follow the contributing guidelines and I have also made one more change as previously requested.
 
 Please, let me know if more changes are needed. I'm always glad to help. :)

**Comment by twalton83** [comment]
Next time you can edit the PR and add commits to the PR instead of closing it :) I’ll check this out more closely soon.

File: javascript/react-js/inputs-and-lists.md
Comment (twalton83) [inline javascript/react-js/inputs-and-lists.md:89]: One last small thing, I realized that we need a root div because we add the Overview component to the App component later on- you can just add that root div back up here so it's more clear. Sorry about that! But thank you for removing the other unnecessary divs! I'll merge this once this is updated. You can just push another commit, instead of opening a new PR.

**Review by twalton83 — CHANGES_REQUESTED** [review]
Made one small edit, realized we needed the root div. After that, I'll merge!

File: javascript/react-js/inputs-and-lists.md
Comment (vinicius77) [reply javascript/react-js/inputs-and-lists.md]: I have just added the root div back. 

Don't be sorry, I'm happy to help. :)

**Review by twalton83 — APPROVED** [review]
Thank you for making these changes!

**Pull request merged**
